mappings/net/minecraft/item/ItemStack.mapping

```diff
@@ -2,132 +2,186 @@ CLASS bcb net/minecraft/item/ItemStack
 	FIELD a EMPTY Lbcb;
 	FIELD b MODIFIER_FORMAT Ljava/text/DecimalFormat;
 	FIELD c LOGGER Lorg/apache/logging/log4j/Logger;
-	FIELD d amount I
-	FIELD e updateCooldown I
+	FIELD d count I
+	FIELD e cooldown I
 	FIELD f item Lbbw;
 	FIELD g tag Lib;
 	FIELD h empty Z
-	FIELD i holdingItemFrame Latf;
-	FIELD j lastCheckedCanHarvestBlock Lbvo;
-	FIELD k lastCheckedCanHarvestResult Z
-	FIELD l lastCheckedCanPlaceBlock Lbvo;
-	FIELD m lastCheckedCanPlaceResult Z
+	FIELD i frame Latf;
+	FIELD j lastDestroyPos Lbvo;
+	FIELD k lastDestroyResult Z
+	FIELD l lastPlaceOnPos Lbvo;
+	FIELD m lastPlaceOnResult Z
+	METHOD <init> (Lbhh;)V
+		ARG 1 item
 	METHOD <init> (Lbhh;I)V
-		ARG 1 container
+		ARG 1 item
+		ARG 2 count
+	METHOD <init> (Lib;)V
+		ARG 1 tag
 	METHOD A getRepairCost ()I
-	METHOD B toTextComponent ()Ljm;
-	METHOD C getUpdateCooldown ()I
-	METHOD D getAmount ()I
+	METHOD B toHoverableText ()Ljm;
+	METHOD C getCooldown ()I
+	METHOD D getCount ()I
 	METHOD E isFood ()Z
 	METHOD F createModifierFormat ()Ljava/text/DecimalFormat;
-	METHOD G updateEmptyFlag ()V
+	METHOD G updateEmptyState ()V
```

Comment:
What was the issue with `flag` ?

Reply:
It updates a cached state used internally to avoid redundant checks. If anything it should be `updateCachedEmptyState` but it feels a little verbose for an internal method.

Reply:
Fair enough